### PR TITLE
Portfolio: Preserve data

### PR DESCRIPTION
**Pull request by sanieni6**
## Portfolio: Preserve data 😎 
In this activity my partner @pmaxy  and I added the next features:
+ Create a Javascript object 🔢 
+ Add the feature to preserve the form data 💾

### CODE REVIEW

File: validation.js [hunk 1]
```diff
@@ -4,6 +4,30 @@ const message = document.createElement('span');
...
+const renderInfo = () => {
+  const storedUserData = JSON.parse(localStorage.getItem('formData'));
+  userName.value = storedUserData.userName;
+  emailInput.value = storedUserData.userEmail;
+  descriptionTextarea.value = storedUserData.message;
+};
+const localStore = (event, value) => {
+  formData[value] = event.target.value;
+  localStorage.setItem('formData', JSON.stringify(formData));
+};
```
Comment (amejid) [inline validation.js:25]: - [ ] I appreciate your effort, but the form data is not getting persisted to the localStorage. Kindly check the function you are using to save the data to localStorage. ✍️

<a href="https://www.loom.com/share/b7a8ce8ca41f4a07bae407a6a514467c">
    <p>Luis Sanchez - 4 February 2023 - Watch Video</p>
    
  </a>

**Review by amejid — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sanieni6** [comment]
## Changes Done
+ Preserve data in the input fields

**Review by huseyinbiyik — APPROVED** [review]
Hi team,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- Form data save the data to localStorage and retrieve it on reload 😎
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝
- Great implementation of the requested changes of previous reviewers 🕸️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**